listenbrainz/webserver/views/stats_api.py

```diff
@@ -818,7 +826,7 @@ def _get_sitewide_entity_list(
     return sorted(result, key=lambda x: x['from_ts'])
 
 
-def _get_country_codes(artist_msids: list, artist_mbids: list) -> List[UserArtistMapRecord]:
+def _get_country_codes(artist_msids: List[Tuple[str, int]], artist_mbids: List[Tuple[str, int]]) -> List[UserArtistMapRecord]:
```

Comment:
the type annotation here seems incorrect, it's a list of dicts.

Reply:
Fixed it.